.github/workflows/pull-request.yml

```diff
@@ -3,7 +3,7 @@ name: Pull Request
 on:
     push:
         branches:
-            - main #todo - remove
```

Comment:
I think this comment was to remove the workflow from running on pushes to main which does make sense given the name of the workflow file at least.

Reply:
Hmm, interesting... we do the same thing in core tho

Reply:
Yeah, we need to fix that in core as well really but it's fine for now.